wpeverest/everest-forms
# Fix - Multi select design issue

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes multi-select design issue in setting page.
![image](https://user-images.githubusercontent.com/66351929/104576579-7d203f00-5680-11eb-931c-cbfd30e3eb9b.png)
![image](https://user-images.githubusercontent.com/66351929/104576602-84474d00-5680-11eb-8c28-192ac38e330a.png)

### How to test the changes in this Pull Request:

1. Checkout to this branch
2. Navigate to payments tab and PDF Submission tab
3. Please check if the design issue still exist or not

### Types of changes:




### Other information:




### Changelog entry

> Fix - Multi select design issue
